test/test-parse.cpp

```diff
@@ -52,13 +52,71 @@ TEST(test_parse, comma_not_allowed_between_class_methods) {
   {
     spy_visitor v;
     padded_string code(
-        u8"class f { constructor() { this._a = false; } ontext(text) { if (this._a) { process.stdout.write(text);}},}"_sv);
+        u8"class f { constructor() { this._a = false; }, ontext(text) { if (this._a) { process.stdout.write(text);}}}"_sv);
     parser p(&code, &v);
     EXPECT_TRUE(p.parse_and_visit_statement(v));
     EXPECT_THAT(v.errors,
                 ElementsAre(ERROR_TYPE_FIELD(
                     error_comma_not_allowed_between_class_methods,
-                    unexpected_comma, offsets_matcher(&code, 104, 105))));
+                    unexpected_comma, offsets_matcher(&code, 44, 45))));
+    EXPECT_THAT(
+        v.visits,
+        ElementsAre("visit_variable_declaration", "visit_enter_class_scope",
```

Comment:
Nit: Suffix each string with a comment for better formatting. See other tests for examples.

Reply:
Thanks, I made this change.